### DISCUSSION TITLE
[Projects Beta] missing multi-select rows in table view for archiving

### DISCUSSION THREAD

**Discussion by jamesacklin** [community/community#56314] [Projects and Issues]
### Select Topic Area

Product Feedback

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

I use the Table view in Projects Beta to do release management and archive issues from the project once they're released. I would greatly appreciate shift-click or some other multi-select in the Table view so I can archive multiple issues at once. At the moment, I have to select one issue at a time and hit the "E" hotkey to archive them.

I'm aware of the [April 6 update](https://github.blog/changelog/2023-04-06-github-issues-projects-april-6th-update/#t-rex-bulk-editing-in-tables) which allows for bulk editing of cells, but I'm unable to act upon the row itself.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by rileybroughten** [comment]
👋 @jamesacklin you can select multiple table items using `shift+click` or `cmd/ctrl+click` to archive multiple items. Check out the video below and let me know if you are still facing issues!

https://github.com/community/community/assets/101840513/25a7e5e0-29d9-4c23-86b0-a53ce1f97827
